packages/ember-toucan-form/package.json

```
@@ -29,19 +29,23 @@
     "test": "echo 'A v2 addon does not have tests, run tests in test-app'",
     "prepack": "rollup --config",
     "format:check": "prettier -c .",
-    "format": "prettier -w ."
+    "format": "prettier -w .",
+    "_syncPnpm": "pnpm sync-pnpm"
   },
   "peerDependencies": {
+    "@crowdstrike/ember-toucan-core": "^0.1.0",
     "@crowdstrike/ember-toucan-styles": "^2.0.1",
     "@ember/test-helpers": "^2.8.1",
     "@glimmer/tracking": "^1.1.2",
+    "ember-headless-form": "1.0.0-beta.1",
```

1. we should probably get a 1.0.0 release out here as well.
2. we should allow for future versions:
```suggestion
    "ember-headless-form": "^1.0.0-beta.1",
```

---

Perfect, just updated! 🎉